### DISCUSSION TITLE
Really dumb question about networking 2 Macs

### DISCUSSION THREAD

**Discussion by toodamnloud** [apohl79/audiogridder#978] [Q&A] [answered]
I'm embarrassed to be asking this because I've had a perfectly functional single crossover cable via Ethernet from Mac server to Mac host for the last year. I recently added a network switch and lost my network settings and everything that worked before fails. I can't get the server that mounts on my desktop at start up to be recognised via AG. My network control panel is green at both ends but the plugin fails to find it. Can anyone tell me how I should configure the Mac system control panel network settings please?

**Comment by toodamnloud** [comment]
How can a second machine on my desktop not be accessible? All file sharing options are on, all file and folder accessability is on, but it can't locate the server.

**Comment by toodamnloud** [comment]
Come on! This is simple stuff, so what am I doing wrong? Somebody here must know. Please help. I have a well spec'd recent Mac with tons of Acustica plugins I can't get at.

**Comment by apohl79** [comment, marked as answer]
If you use a crossover cable, you have to connect the two computers directly. With a switch in between you need a regular cable. You can try to manually set the IP addresses of your two computers. Make sure to pick a separate local network range (for example 192.168.10.1 for computer 1 and  192.168.10.2 for the second one. And set the network mask to 255.255.255.0.)

**Comment by toodamnloud** [comment]
Thank you very much for your help. This worked by setting the ip addresses separately and manually but I also needed to set the network order, placing Ethernet ahead of the wireless settings.
Anyway, thanks for a great product! Audiogridder is fantastic.
If you use a crossover cable, you have to connect the two computers directly. With a switch in between you need a regular cable. You can try to manually set the IP addresses of your two computers. Make sure to pick a separate local network range (for example 192.168.10.1 for computer 1 and 192.168.10.2 for the second one. And set the network mask to 255.255.255.0.)
